apps/vaults/components/details/actions/QuickActionsTo.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {useActionFlow} from '@vaults/contexts/useActionFlow';
+import {useSolver} from '@vaults/contexts/useSolver';
+import {toAddress} from '@yearn-finance/web-lib/utils/address';
+import {formatCounterValue} from '@yearn-finance/web-lib/utils/format.value';
+import {Dropdown} from '@common/components/TokenDropdown';
+import {useTokenPrice} from '@common/hooks/useTokenPrice';
+import {formatPercent} from '@common/utils';
+
+import type {ReactElement} from 'react';
+import type {TDropdownOption} from '@common/types/types';
+
+function	VaultDetailsQuickActionsTo(): ReactElement {
```

Comment:
This whole file looks very similar to `VaultDetailsQuickActionsFrom.tsx`, could we make a generic `VaultDetailsQuickActions` that takes a prop "from" | "to"? Or are they too diff that's making it more confusing?

Reply:
All good, at a glance they just looked very similar. Let's keep as is, thanks!